eos/form-factors/observables.cc

```diff
@@ -597,25 +559,24 @@ namespace eos
                 make_form_factor_adapter("B->K^*::F_long_T(q2)",
                         &FormFactors<PToV>::f_long_T, std::make_tuple("q2")),
 
```

Comment:
This is not removed, maybe this is a display issue or you missed one?

Reply:
This is ok, what I removed is the ratio, not the adapter.